set.mm

```diff
@@ -467607,6 +467823,274 @@ real number multiplication operation (this has to be defined in the main
+        PVYLUHVWOVWSVWPUYJXTXDVWOVYKUYAVWSVWOUYAUYJYBZVULVUIYBZUYJVUIVHZXKUHZVY
+        KUYAUHVUGVYNVUMVUGUYAVWEYBZVYNVUGVVOVYRVUGVVNVVOVVQVVRVPZDUYAUVPVPVUGVW
+        EUYJUYAVUGVVJEUIUJVUGUYAVUCVIZVVJEUHZVUGVUBVUCUYAVVPUVQVYTUYAXAVIZWUAXA
+        VSUVRVFURZVEYCVEXAYBVYTWUBWUAWFUVSUVTXAWUCVEYDXAEUYAVEUWAUWBWPVPZUWCZUW
+        DXOXLZVWOVUMVYOVXKVUIVUKVULYAVPZVUGVYQVUMVUGVXFVYPXKVUGVUHUYJVUIVUGVUHV
+        WEUYJVWFWUEXSZUWEVXGVUGVXLXDXBXLZUYJVUIUYAVULYEYFYGVWOVWSVUJUPZUYAUNZUY
+        AUMZUYBUNZUYAUNZVYMUYKVWOWUKWUMVUJUPZUYAUNZWUNVWOWUJWUOUYAVWOWUJVWRGVUJ
+        UPZUNZWUOVWRGVUJXTVWOWUOWULUYBVUJUPZUNZVWRVUJUPZWUQUNZWURWUOWUTUHVWOWUL
+        UYBVUJXTXDVWOWULWVAWUSWUQVWOWVAWULVULUMZURZVUJUPZWULVWRWVDVUJUYAVULUWFZ
+        YHVWOWULVUJYBZWVCVUKYBZVXHWVEWULUHVWOVXEVUJVUHWULULZWVGVXJVUHVUJUYAXPZV
+        UJVUHWULYAWJZVWOVUMVUKVUIWVCULZWVHVXKVUIVUKVULXPZVUKVUIWVCYAWJZVXMVUJVU
+        KWULWVCYEYFUWGVWOUYBGVUJVUFUYCVUMWDZYIZYMVWOWUQVKZVUJXCWVBWURUHVWOWVQVU
+        JVUJVWOWVQUYAUTUQUKZWULUNZVKZVUJVWOWUQWVSVWOWUSWUQWVSWVPVWOIDVOUYAPVUGV
+        VBVUMVWLXLZVUGVVOVUMVYSXLZVXIUWHXBYJVWOVUJVUJWVSULZVUJVUJWVSUWIWVTVUJUH
+        VWOVUHVUJWVRULZWVIWWCVWOVVOVVTUTXJVIWWDWWBVXIVWOUWJDUTUYAUWKYFVWOVXEWVI
+        VXJWVJVPVUJVUHVUJWVRWULXRWTVUJVUJWVSUWLVUJVUJWVSUWMWJXSVUJUWNZYKVWRWUQV
+        UJYLVPUWOYNYOVUJVUJXCZWUPWUNUHVWOWWEWUMUYAVUJYLVNXSWWFWUKVYMUHVWOWWEVWS
+        UYAVUJYLVNVWOCDEHIKVOUYAMNOPWWAWWBVXIVUGWUAVUMWUDXLUWPUWQYPVWOVXQVWSVUL
+        UNZUYMVWOVXQVWSVWPUYLUPZUNWWGVWSVWPUYLXTVWOWWHVULVWSVWOVWPVUIUPZWWHVULV
+        WOVUIUYLVWPVWOVUIUYDUYJVJZUYLVWOVUHUYJUYDVUGVUHUYJUHVUMWUHXLUWRAWWJUYLU
+        HZVUEUYCVUMAVYHWWKTEUIKUWSVPXQXSZYQVWOVYNVYOVYQWWIVULUHWUFWUGWUIUYJVUIU
+        YAVULYRYFXBYGYNVWOWVCUOVUKUPZUNZVULUNZVWSVUKUPZVULUNZUYMWWGVWOWWNWWPVUL
+        VWOVWRVUKUPZGVUKUPZUNZWWNWWPVWOWWRWVCWWSWWMVWOWWRWVDVUKUPZWVCVWRWVDVUKW
+        VFYHVWOWVGWVHVXHWXAWVCUHWVKWVNVXMVUJVUKWULWVCYRYFYNVWOUYBVUKUPWWSWWMVWO
+        UYBGVUKWVOYIVWOIDVOUYAPWWAWWBVXIUWTXBZYMVWOWWTVWRWWSUNZWWPVWOWWSVKZVUKX
+        CWWTWXCUHVWOWXDWWMVKZVUKVWOWWSWWMWXBYJWXEVUKVUKUXAUXCYKVWRWWSVUKYLVPVWR
+        GVUKXTUXDXBYOVWOWWOWVCVULUNZUOVUIUPZUYMVWOWWNWVCVULVWOVUKVUIWVCYCZWWNWV
+        CUHVWOVUMWVLWXHVXKWVMVUKVUIWVCYSWJVUKVUIWVCUXBVPYOVWOVUMWXFWXGUHVXKVUIV
+        UKVULUXEVPVWOVUIUYLUOWWLYQYPVWOVULVKVUKXCZWWQWWGUHVWOVUMVUIVUKVULYCWXIV
+        XKVUIVUKVULYSVUIVUKVULUXFWJVWSVULVUKYLVPUXGXSUXHXSUYPVWQVXAWFLVWPUYAVUL
+        UCYTUEYTUXIUYEVWPUHZUYFVWQUYOVXAUYDDUYEVWPUXJWXJUYIVWTUYNWXJUYHVWSUYEVW
+        PWXJUYGVWRGUYEVWPUXKYOWXJWKYMUXLUXMUXNWTUXOAUCVUBVUCGIUCIUXPAVVBVUBBIYC
+        IVUBYBUAUDBIDHVOPNQUXQVUBBIYDWJAGCIVUCVGUBMUXRUXSUXT $.
+
+      $d .+ p q t $.  $d .- p q t $.  $d B p q t $.  $d D p q t $.  $d D u w $.
+      $d M p q t $.  $d M u $.  $d N p q t $.  $d P p q t $.  $d Q p q t $.
+      $d Q u $.  $d T p q t $.  $d p ph q t $.
+      cycpmconjs.t $e |- ( ph -> T e. C ) $.
+      $( All cycles of the same length are conjugate in the permutation group.
```

Comment:
Better to write "in the symmetric group", since clearly cycles of the same length need not be conjugate in an arbitrary permutation group.

Reply:
Yes! I've fixed that, and also corrected the "order of the group" (number of permutations in the group) to "degree of the group", (the number of elements in the underlying set).

Reply:
I cannot find the definition of "degree of the group" as "the number of elements in the underlying set" in set.mm (and in [Lang} neither). In Wikipedia, however, I found: "The alternating group on a set of n elements is called the alternating group of degree n, ..." (https://en.wikipedia.org/wiki/Alternating_group). But the term "degree" is defined for "symmetric groups" only, not for arbitrary groups.

If this term is used in set.mm, it should be defined somewhere (at least in the commt for ~cyc3conja where it is used).

Reply:
Or simply:
> All 3-cycles are conjugate in the alternating group A_n for n>= 5.

or
> All 3-cycles are conjugate in the alternating group A_n on n >= 5 elements.

Reply:
> But the term "degree" is defined for "symmetric groups" only, not for arbitrary groups.

It is defined for permutation groups, see for instance https://en.wikipedia.org/wiki/Permutation_group#Basic_properties_and_terminology.

Reply:
I originally wrote _order_, which was wrong, and which I corrected into _degree_ which is indeed the correct term.
However it's true that it's not defined in `set.mm`, so I'll go with @benjub's proposal.